### ISSUE TITLE
Pamphlet option does not seem to work with absolute pathing

### ISSUE THREAD

**Issue by OlavStornes** [na--/ebook-tools#10]
Using the docker image - 

It seems whenever i attempt to sort pamphlets with an absolute path, the folder structure becomes weird. Almost like the filename includes the old path. Note the double slash in the TO-line

```
OK:	/vault/Literature/test/example_pamphlet.pdf

TO:	/vault/Literature/Pamphlets//vault/Literature/test/example_pamphlet.pdf
```

The command i'm using at the moment:

`organize-ebooks.sh -o=/vault/Literature/Books -owi -ofu=/vault/Literature/Uncertain -ofp=/vault/Literature/Pamphlets -ofc=/vault/Literature/z_corrupted --keep-metadata /vault/Literature/test`

A small hotfix that i have found working is to change the current directory to my unsorted folder and use the current folder as an argument instead. Then it outputs to `"/vault/Literature/Pamphlets/./Proofmarks/example_pamphlet.pdf"` instead, which makes it work as intended at least.

**Comment by keithras** [comment]
Yeah I'm noticing the same thing on the current latest version. I thought that this was quite possible a feature, and not a bug. Maybe the software wants to preserve the original folder structure to keep those together?

**Comment by OlavStornes** [comment]
It could definetly be an intended feature. If that was the case then all is well, but i'm a bit unsure on this due to the strange output.

**Comment by ndc168** [comment]
Somehow the "Pamphlets" folder is special (centralized) compared to other output folder args...

If you specify other output folders as sub-folders, the sub-foldering would be correct. But the Pamphlets path would be repeated (as complained above), as if pamphlets shall be sent to a centralized location..

Can this be corrected? Not everyone use Calibre and some have a custom folder structure to maintain.
